languages/java/exercises/concept/classes/src/test/java/RemoteControlCarTest.java

```diff
@@ -0,0 +1,120 @@
+import org.junit.Assert;
+import org.junit.Ignore;
+import org.junit.Test;
+
+public class RemoteControlCarTest
+{
```

Comment:
normally in Java left curly bracket is on the same line of the class name
so 
```
class Car {
}
```
This is valid for all the code in this page
This is valid for block of code too